Makefile

```diff
@@ -246,7 +246,7 @@ ifeq ($(AUTOSPAWN), true)
 	${MAYBE_ENVIRONMENT_EXEC} $(CONTAINER_TOOL)-compose up -d dependencies-kafka
 	sleep 5 # Many services are very lazy... Give them a sec...
 endif
-	cargo test --no-default-features --features kafka-integration-tests ::kafka:: -- --nocapture
+	cargo test --no-default-features --features kafka-integration-tests --features rdkafka-plain ::kafka:: -- --nocapture
```

Comment:
```suggestion
	cargo test --no-default-features --features kafka-integration-tests rdkafka-plain ::kafka:: -- --nocapture
```

Reply:
No, this will not work:
```bash
$ cargo test --no-default-features --features kafka-integration-tests rdkafka-plain ::kafka:: -- --nocapture
error: Found argument '::kafka::' which wasn't expected, or isn't valid in this context

USAGE:
    cargo test --features <FEATURES>... --no-default-features

For more information try --help
```

Reply:
Sorry, you're right. :) You need to do it your way or wrap it all in a string 

```
--features "kafka-integration-tests  rdkafka-plain"
```